src/main/java/fr/techad/edc/popover/internal/swing/builder/ContextualContentComponentBuilderImpl.java

```diff
@@ -29,16 +34,22 @@
     private final Popover popover;
     private ContextItem contextItem;
     private Color backgroundColor = Color.WHITE;
+    private ErrorBehavior errorBehavior;
+    private final JLabel friendlyMessage = new JLabel("Contextual help is coming soon.");
+    private final JLabel errorMessage = new JLabel("<html>An error occurred when fetching data ! <br/> Check the brick keys provided to the EdcHelp component.</html>");
+    private HttpReaderImpl httpReader;
+    private String languageCode = "en";
     private boolean enableRelatedTopics = true;
     private Color popoverSectionTitleColor = Color.BLACK;
     private Font popoverSectionTitleFont = new Font("Dialog", Font.BOLD, 12);
     private boolean enableArticle = true;
 
     @Inject
-    public ContextualContentComponentBuilderImpl(EdcClient edcClient, OpenUrlAction openUrlAction, Popover popover) {
+    public ContextualContentComponentBuilderImpl(EdcClient edcClient, OpenUrlAction openUrlAction, Popover popover, HttpReaderImpl httpReader) {
```

Comment:
First: We inject the interface not the implementation
Second: This class is encapsulated in EdcClient and should never be used (and read the file for each popover is overkill)
Third: httpReader is never used, so removed it.